docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,418 @@
+
+### Bi-direction connection operator (:<>=)
+
+For connections where you want 'bulk-connect-like-semantics' where the aligned fields are driven producer-to-consumer and flipped fields are driven consumer-to-producer, use `:<>=`.
+
+```scala mdoc:silent
+class Example1 extends RawModule {
+  val foo = IO(Flipped(new MixedAlignmentBundle))
+  val bar = IO(new MixedAlignmentBundle)
+  bar :<>= foo
+}
+```
+
+This generates the following Verilog, where the aligned members are driven `foo` to `bar` and flipped members are driven `bar` to `foo`:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example1())
+```
+
+### Aligned connection operator (:<=)
+
+For connections where you want the aligned-half of 'bulk-connect-like-semantics' where the aligned fields are driven producer-to-consumer and flipped fields are ignored, use `:<=` (the "aligned connection").
+
+```scala mdoc:silent
+class Example2 extends RawModule {
+  val foo = IO(Flipped(new MixedAlignmentBundle))
+  val bar = IO(new MixedAlignmentBundle)
+  foo.b := DontCare // Otherwise FIRRTL throws an uninitialization error
+  bar :<= foo
+}
+```
+
+This generates the following Verilog, where the aligned members are driven `foo` to `bar` and flipped members are ignored:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example2())
+```
+
+### Flipped connection operator (:>=)
+
+For connections where you want the flipped-half of 'bulk-connect-like-semantics' where the aligned fields are ignored and flipped fields are assigned consumer-to-producer, use `:<=` (the "flipped connection", or "backpressure connection").
+
+```scala mdoc:silent
+class Example3 extends RawModule {
+  val foo = IO(Flipped(new MixedAlignmentBundle))
+  val bar = IO(new MixedAlignmentBundle)
+  bar.a := DontCare // Otherwise FIRRTL throws an uninitialization error
+  bar :>= foo
+}
+```
+
+This generates the following Verilog, where the aligned members are ignore and the flipped members are driven `bar` to `foo`:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example3())
+```
+
+### Coercing mono-direction connection operator (:#=)
+
+For connections where you want to every producer field to always drive every consumer field, regardless of alignment, use `:#=` (the "coercion connection"). This operator is useful for initializing wires whose types contain members of mixed alignment.
+
+```scala mdoc:silent
+import chisel3.experimental.BundleLiterals._
+class Example4 extends RawModule {
+  val bar = Wire(new MixedAlignmentBundle)
+  dontTouch(bar) // So we see it in the output verilog
+  bar :#= (new MixedAlignmentBundle).Lit(_.a -> true.B, _.b -> true.B)
+}
+```
+
+This generates the following Verilog, where all members are driven `foo` to `bar`, regardless of alignment:
```

Comment:
there is no foo in the above. can you assign foo to the temporary? Can we call it "literal"?